dashboard/test/ui/features/step_definitions/droplet_steps.rb

```diff
@@ -43,6 +43,9 @@
 end
 
 When /^I ensure droplet is in text mode$/ do
+  steps <<-STEPS
+    And I wait to see "#show-code-header"
+  STEPS
```

Comment:
nit: this can just be `steps 'And I wait to see "#show-code-header"'`, here and below